static/app/views/admin/installWizard/index.tsx

```diff
@@ -93,55 +102,33 @@ export default class InstallWizard extends DeprecatedAsyncComponent<
       }
     });
     return data;
-  }
-
-  render() {
-    const version = ConfigStore.get('version');
-    return (
-      <SentryDocumentTitle noSuffix title={t('Setup Sentry')}>
-        <Wrapper>
-          <Pattern />
-          <SetupWizard>
-            <Heading>
-              <span>{t('Welcome to Sentry')}</span>
-              <Version>{version.current}</Version>
-            </Heading>
-            {this.state.loading
-              ? this.renderLoading()
-              : this.state.error
-                ? this.renderError()
-                : this.renderBody()}
-          </SetupWizard>
-        </Wrapper>
-      </SentryDocumentTitle>
-    );
-  }
-
-  renderError() {
-    return (
-      <Alert type="error" showIcon>
-        {t(
-          'We were unable to load the required configuration from the Sentry server. Please take a look at the service logs.'
-        )}
-      </Alert>
-    );
-  }
-
-  renderBody() {
-    return (
-      <ApiForm
-        apiMethod="PUT"
-        apiEndpoint={this.getEndpoints()[0]![1]!}
-        submitLabel={t('Continue')}
-        initialData={this.getInitialData()}
-        onSubmitSuccess={this.props.onConfigured}
-      >
-        <p>{t('Complete setup by filling out the required configuration.')}</p>
-
-        {this.renderFormFields()}
-      </ApiForm>
-    );
-  }
+  };
+
+  const version = ConfigStore.get('version');
+  return (
+    <SentryDocumentTitle noSuffix title={t('Setup Sentry')}>
+      <Wrapper>
+        <Pattern />
+        <SetupWizard>
+          <Heading>
+            <span>{t('Welcome to Sentry')}</span>
+            <Version>{version.current}</Version>
+          </Heading>
+          <Form
+            apiMethod="PUT"
+            apiEndpoint={'/internal/options/?query=is:required'}
+            submitLabel={t('Continue')}
+            initialData={getInitialData()}
```

Comment:
does this component work well enough to fill in the initial data when the data loads in later?